spec/api-browser-window-spec.js

```diff
@@ -1455,7 +1455,10 @@ describe('browser-window module', function () {
 
   describe('window.webContents.executeJavaScript', function () {
     var expected = 'hello, world!'
+    var expectedErrorMsg = 'woops!'
     var code = '(() => "' + expected + '")()'
+    var asyncCode = '(() => new Promise(r => setTimeout(() => r("' + expected + '"), 500)))()'
```

Comment:
Maybe we should switch these two new code strings and the existing `code` variable to use template strings for clarity.